### DISCUSSION TITLE
Octolamp build – ABS & LED Splice

### DISCUSSION THREAD

**Discussion by tgolla** [martinwoodward/octolamp#23] [Show and tell]
With my build I decided to print it in ABS. This was my first time printing a light color (white) in ABS, which became problematic.  The base came out fine, but light colors in ABS can have a tendance to get small scorch marks.  This is not a problem if you have no intention of shining light through the material as these can be scrapped off the surface.  But when imbedded in a layer with light shining through these create a less than aesthetic effect.  I also tried HIPS which had the same issue and is also less translucent. The end solution was an ABS base in black and the reflector/cover in white PLA.

One change I did make was to split and solder the LEDs with 1” of lead wire such that I could route the inner circle of LEDs in the same clockwise direction as the outer LEDs. This allows effects to spin in the same direction and should also make segmenting easier.

![20230608_172725](https://github.com/martinwoodward/octolamp/assets/14251985/f9b8d2fd-fb28-406c-a3e3-971e01a5dbff)

**Comment by martinwoodward** [comment]
Great to know, I've never been brave enough to try ABS. Glad the tolerances are flexible enough to do the material combo

That's an exceptionally neat wiring job. Great work!
